mem-12-q.xml

```diff
@@ -6145,7 +6203,7 @@ Maltz seemed reluctant to talk about what exactly this is.[2]</column>
       <column name="notes_de"></column>
       <column name="notes_fa"></column>
       <column name="notes_sv"></column>
-      <column name="notes_ru"></column>
+      <column name="notes_ru">Также подходит для обозначения процесса регистрации учётной записи {mab qI':sen@@mab:n, qI':v}.[2]</column>
```

Comment:
This entry and note has changed. Please sync up with https://github.com/De7vID/klingon-assistant-data/tree/master (you're ~160 commits behind).